db.go

```diff
@@ -90,13 +94,12 @@ type LdapUser struct {
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 // Authenticator/Userstore that simply stores identity/passwords in memory
-type dummyUserStoreAndAuth struct {
+type dummyUserStore struct {
 	users     map[UserId]*dummyUser
 	usersLock sync.RWMutex
 }
 
-type dummyLdapUserStoreAndAuth struct {
-	imqsUsers map[UserId]*dummyUser
+type dummyLdap struct {
```

Comment:
The LdapUser type is only ever used by dummyLdap, so it should rather be named as such, and not exported from authaus.